frontend/src/styles/Moment.css

```diff
@@ -26,8 +37,49 @@
     position: relative;
     float: left;
     width:  400px;
-    height: 200px;
+    height: 320px;
     background-position: 50% 50%;
     background-repeat:   no-repeat;
     background-size:     cover;
 }
+
```

Comment:
For all of these rotations, the animation should be made smooth for when the element returns to its original state. At the moment, as soon as the mouse leaves, it snaps back and looks choppy.

Reply:
Okay noted, I know how to adjust that.

Reply:
:eyes: